src/map/skill.cpp

```diff
@@ -5107,6 +5108,15 @@ int skill_castend_damage_id (struct block_list* src, struct block_list *bl, uint
 	map_freeblock_lock();
 
 	switch(skill_id) {
+	case HFLI_MOON:	//[orn]
+	case HFLI_SBR44:	//[orn]
+	{
+		if (hd)
+			skill_blockhomun_start(hd, skill_id, skill_get_cooldown(skill_id, skill_lv));
+		if (master_sd)
+			skill_blockpc_start(master_sd, skill_id, skill_get_cooldown(skill_id, skill_lv));
```

Comment:
```suggestion
		if (hd != nullptr) {
			skill_blockhomun_start(hd, skill_id, skill_get_cooldown(skill_id, skill_lv));
			if (hd->master != nullptr)
				skill_blockpc_start(hd->master, skill_id, skill_get_cooldown(skill_id, skill_lv));
		}
```
Instead of having to define `master_sd` everywhere, couldn't you adjust it to like this? I don't think this change would apply to pets, elemental, etc that are pulled from `battle_get_master()`.

Reply:
good, previously battle_get_master was also used for the homunculus skill too, I changed it according to your suggestion

Reply:
I completely forgot to mention this in my previous comment, but we will probably need to wrap this in the RENEWAL ifdef to make sure we don't break pre-renewal.

Reply:
> I completely forgot to mention this in my previous comment, but we will probably need to wrap this in the RENEWAL ifdef to make sure we don't break pre-renewal.

I added #ifdef on homunculus base skills cooldown now, I think everything else is in the appropriate #ifdef RENEWAL blocks, let me know if I missed something